client/utils/rest.go

```diff
@@ -10,3 +16,30 @@ func WriteErrorResponse(w *http.ResponseWriter, status int, msg string) {
 	(*w).WriteHeader(status)
 	(*w).Write([]byte(msg))
 }
+
+// WriteGasEstimateResponse prepares and writes an HTTP
+// response for transactions simulations.
+func WriteSimulationResponse(w *http.ResponseWriter, gas int64) {
```

Comment:
Do you need to pass this as a pointer?

Reply:
(also for the above function?)

Reply:
No, I don't. Amended, thanks for pointing it out!